portal/server/helpers/constructPayload.test.js

```diff
@@ -88,8 +88,25 @@ describe('Unit test cases for constructPayload method', () => {
       password: 'AbC!23456',
       passwordConfirm: 'AbC!23456',
     };
-    const returned = constructPayload(mockBody, payloadProperties, false);
+    const returned = constructPayload(mockExtraBody, payloadProperties, [], false);
 
     expect(expected).toEqual(returned);
   });
+
+  it('Should delete currency with empty values', () => {
+    const expected = {
+      currentCurrencyValue: '',
+      newCurrencyValue: 'value',
+      anotherCurrenyValue: '',
+    };
+    const unsetIfPropertyIsEmpty = ['currentCurrencyValue', 'newCurrencyValue', 'anotherCurrenyValue'];
+    // const returned = constructPayload(mockBody, payloadProperties, false);
```

Comment:
Purpose of this unit test file is to test function `constructPayload`, but you are not calling this function, please uncomment line 103.

You can create new `mockBody`, it is your `input`. Value for `expected` can be object with single value, it is your `expected output`.

```
const mockBody = {
      currentCurrencyValue: '',
      newCurrencyValue: 'value',
      anotherCurrenyValue: '',
};
const expected = {
      newCurrencyValue: 'value'
};

Reply:
Uncommented  line 103 and created new `mockBodyWithEmptyValues`, `payloadPropertiesForEmptyValues`